go.mod

```diff
@@ -9,3 +9,5 @@ require (
 	github.com/pkg/errors v0.8.1
 	github.com/zchee/go-vmnet v0.0.0-20161021174912-97ebf9174097
 )
+
+replace github.com/code-ready/machine => github.com/guillaumerose/machine v0.0.0-20200929195148-b45acbbd8a57
```

Comment:
Unwanted change.

Reply:
Yes, I need this https://github.com/code-ready/machine/pull/43 to be merge before.